src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/input/Microsoft.Android.Sdk.BundledVersions.in.targets

```diff
@@ -0,0 +1,34 @@
+<!--
+***********************************************************************************************
+Microsoft.Android.Sdk.BundledVersions.targets
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+***********************************************************************************************
+-->
+<Project>
+  <PropertyGroup>
+    <AndroidNETSdkVersion>@AndroidPackVersionLong@</AndroidNETSdkVersion>
+    <XamarinAndroidVersion>@AndroidPackVersionLong@</XamarinAndroidVersion>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetPlatformVersion)' == '31.0' ">
+    <_AndroidRuntimePackId>32</_AndroidRuntimePackId>
+  </PropertyGroup>
+  <PropertyGroup>
+    <_AndroidTargetingPackId>$(TargetPlatformVersion.TrimEnd('.0'))</_AndroidTargetingPackId>
```

Comment:
If you address the comment above, you might put this property earlier so you can check against `31` without the `.0`.

Reply:
Updated